mdai/__init__.py

```diff
@@ -1,6 +1,6 @@
 """MD.ai Python client library."""
 
-__version__ = "0.8.0"
+__version__ = "0.8.1"
```

Comment:
We'll bump the version in a separate commit on the main branch: https://github.com/mdai/mdai-client-py/commit/59595a023e73571f36e1c7951325476d73ef3823